### PR TITLE
New Design for Flowey's Time Machine

### PR DESCRIPTION
**Pull request by ardasoyturk**
Hello there, [@tarikcoskun](https://github.com/tarikcoskun) and I made a new design with Undertale fonts for your time machine. We'll be glad if you want to merge it to your repository.

https://floweystimemachine.github.io

image
image

### CODE REVIEW

**Comment by ardasoyturk** [comment]
Also, we didn't put the "Your Presets" section to the new one because we couldn't decide where to put it. So if you are going to merge it, you might want to add the "Your Presets" section.

**Comment by crumblingstatue** [comment]
This looks pretty good, thank you!
I'll review it over the weekend and see if I can merge it!

File: help.html [hunk 1]
```diff
@@ -4,62 +4,75 @@
...
+                Just delete these files using your file manager.
+                If you are using the Steam version of the game, these files might be backed up and restored by
+                the game on startup.
+                In that case, you should delete the Steam data for UNDERTALE in addition to the system_ files,
+                but I have no information on how to do this.
+            </p>
+
+        </div>
+        <footer class="fixed bottom-0 w-full p-2 mt-2 text-center md:static" style="background: #383838; text-shadow: 0 2px #000">
+            This time machine is a modified version of <a href="https://github.com/crumblingstatue"
```
Comment (crumblingstatue) [inline help.html]: This part is kind of unnecessary if you want to merge this code into the main repository, haha

File: main.js [hunk 2]
```diff
@@ -605,7 +607,7 @@ function updatePersistentDataForm(iniobj) {
         document.getElementById("ini-dodged-all-special-thanks").checked = false;
     }
     if (iniobj.fun) {
-        document.getElementById("ini-fun").value = parseInt(iniobj.fun.trim());
+        //document.getElementById("ini-fun").value = parseInt(iniobj.fun.trim());
```
Comment (crumblingstatue) [inline main.js]: What happened to this? Why is it commented out? Is there a replacement functionality?

File: main.js [hunk 3]
```diff
@@ -652,10 +654,10 @@ function updateIniFromForm(ini) {
             ini.reset.s_key = "0";
         }
     }
-    var fun = parseInt(document.getElementById("ini-fun").value);
+    /*var fun = parseInt(document.getElementById("ini-fun").value);
```
Comment (crumblingstatue) [inline main.js]: Same here

File: main.js [hunk 4]
```diff
@@ -957,7 +960,7 @@ function start() {
         var name = document.getElementById("builtinpresetselect").value;
         loadPreset(name);
     }, false);
-    function saveUserPreset(name) {
+    /*function saveUserPreset(name) {
```
Comment (crumblingstatue) [inline main.js]: I would prefer this functionality to be restored too before I merge.

**Review by crumblingstatue — CHANGES_REQUESTED** [review]
Overall, I would like no loss of functionality with the new look, if possible.
In the meantime, feel free to refer people to your version.

File: main.js (hunk 2 above)
Comment (tarikcoskun) [reply main.js]: I tried out the system several times and I didn't see its value change neither in our project nor in your project. Same thing with "sav-kills" value. Also I needed some space to be able to make labels as readable as possible. I can add them back if they are needed for the system :)

**Comment by tarikcoskun** [comment]
Got everything done, removed some of our useless stuff, guess ready to go!

**Review by crumblingstatue — APPROVED** [review]
Alright, looks good to me.
Thank you for your contribution!

**Comment by KockaAdmiralac** [comment]
Is `manifest.appcache` needed now that it has been removed from the `<html>` tag on `index.html`?

**Comment by ardasoyturk** [comment]
@crumblingstatue, I noticed that fonts are broken because of the Github Pages' system. You might want to remove "/" from the @font-face section of the style.css file to make the fonts work. Unfortunately, we couldn't notice it when we opened this PR.

image
image

**Comment by crumblingstatue** [comment]
> Is `manifest.appcache` needed now that it has been removed from the `<html>` tag on `index.html`?

If you require it, you can submit a pull request to add it back to the html tag. I'm impartial on it.

**Pull request merged**